src/components/buoy-telemetry/BuoyMap.vue

```diff
@@ -44,12 +44,25 @@ const props = defineProps<{
   variable: string;
 }>();
 
+const varDomain = {
+  chlorophyllDomain: [0, 45],
+  // add other vars here as needed
+};
+const sizeRange = {
+  chlorophyllSizeRange: [0.2, 0.65],
+};
+const colorRange = {
+  chlorophyllColorRange: ["#93C572", "#32CD32", "#355E3B"],
+};
+
 const annotatedSamples = computed(() => {
-  const domain = [0, 45];
-  const sqrtScale = scaleSqrt().domain(domain).range([0.2, 0.65]);
+  const domain = varDomain.chlorophyllDomain;
```

Comment:
then here you could deconstruct like this

```js
const { chlorophyll: {varDomain, sizeRange, colorRange} = config
```

Reply:
In typescript, I think something is missing. I get this error and I'm not sure what `(property)` should be? 
`(property) chlorophyll?: {
    chlorophyll: {
        varDomain: number[];
        markerSize: number[];
        markerColors: string[];
    };
}
A destructuring declaration must have an initializer.ts(1182)`